app/services/translation_service/store/translation.rb

```diff
@@ -0,0 +1,201 @@
+module TranslationService::Store::Translation
+
+  CommunityTranslationModel = ::CommunityTranslation
+
+  Translation = EntityUtils.define_builder(
+    [:translation_key, :mandatory, :string],
+    [:locale, :mandatory, :string],
+    [:translation])
+
+
+  module_function
+
+  # Create translations
+  # Format for translation_groups:
+  # [ { translation_key: nil // optional key - if defined it will override previous translations
+  #   , translations:
+  #     [ { locale: "en-US"
+  #       , translation: "Welcome"
+  #       }
+  #     , { locale: "fi-FI"
+  #       , translation: "Tervetuloa"
+  #       }
+  #     ]
+  #   }
+  # ]
+  def create(community_id:, translation_groups: [])
+    if translation_groups.empty?
+      msg = "You must specify 'translation_groups' as an array of hash-objects containing translation_key and array of translations - like translation_groups: [ { translation_key: nil, translations: [ { locale: 'en-US' , translation: 'Hi!'}, { locale: 'fi-FI', translation: 'Moi!'}]}]"
+      raise ArgumentError.new(msg)
+    end
+
+    translation_groups
+      .map { |group|
+        enforced_key = group[:translation_key]
+        key = Maybe(enforced_key).or_else(gen_translation_uuid(community_id))
+
+        translations = group[:translations].map { |translation|
+
+          translation_hash = {
+            community_id: community_id,
+            translation_key: key,
+            locale: translation[:locale],
+            translation: translation[:translation]
+          }
+          if enforced_key.present?
+            update_translation(translation_hash)
+          else
+            create_translation(translation_hash)
+          end
+        }
+
+        { translation_key: key,
+          translations: translations
+        }
+      }
+  end
+
+  # Get translations
+  # Format for params:
+  # {community_id: 1, translation_keys: ["aa", "bb", "cc"], locales: ["en", "fi-FI", "sv-SE"], options: {}}
+  def get(community_id:, translation_keys: [], locales: [], options: {})
```

Comment:
:use_fallbacks is the only option so it would be more descriptive to just have that as a keyword param with default value. This makes it clear both at the call site as well as here when reading the code.